src/components/typography.js

```diff
@@ -38,6 +38,9 @@ export const Paragraph = styled.p(({ align, width, maxWidth }) => `
   width: ${ width };
   max-width: ${ maxWidth };
   margin: 1rem auto 2rem auto;
+  & .bold {
```

Comment:
see comments below, but .. while this does work, i don't think adding functionality (and thus complexity) to this component is the cleanest approach